src/components/Search/CompareOverTime.tsx

```diff
@@ -21,32 +21,29 @@ const stringsNew =
   Strings.components.searchDefault.overTime.collapsed.revisions;
 
 interface CompareWithTimeProps {
-  isEditable: boolean;
-  baseRevs: Changeset[];
+  hasNonEditableState: boolean;
   newRevs: Changeset[];
 }
 
-function CompareOverTime({ isEditable }: CompareWithTimeProps) {
+function CompareOverTime({
+  hasNonEditableState,
+  newRevs,
+}: CompareWithTimeProps) {
+  const { enqueueSnackbar } = useSnackbar();
   const [expanded, setExpanded] = useState(false);
   const [timeRangeValue, setTimeRangeValue] = useState(
     86400 as TimeRange['value'],
   );
 
+  const [inProgressRevs, setInProgressRevs] = useState<Changeset[]>(newRevs);
+
+  const [repository, setRepository] = useState('try' as Repository['name']);
+
   const mode = useAppSelector((state) => state.theme.mode);
   const styles = CompareCardsStyles(mode);
   const dropDownStyles = SearchStyles(mode);
   const search = useAppSelector((state) => state.search);
   const searchResults = search.new.searchResults;
-  //temporary hash to hide the component until functionality is complete
-  const location = useLocation();
-  const hash = location.hash;
-
-  //temporary styles to hide the component based on hash
-  const containerStyles = {
-    container: style({
-      display: hash === '#comparetime' ? 'block' : 'none',
-    }),
-  };
```

Comment:
should we keep this toggle until the loader is done?

Reply:
Okay, I can leave it and remove the disable rule to the button until the loader part is done.